tests/Microsoft.Identity.Web.Test/TestDistributedCache.cs

```diff
@@ -10,13 +10,23 @@ namespace Microsoft.Identity.Web.Test
 {
     public class TestDistributedCache : IDistributedCache
     {
-        public readonly ConcurrentDictionary<string, byte[]> dict = new ConcurrentDictionary<string, byte[]>();
+        internal readonly ConcurrentDictionary<string, Entry> _dict = new ConcurrentDictionary<string, Entry>();
```

Comment:
Make it a property if it's public / internal. Or create a "Count" method and keep this private.